.travis.yml

```diff
@@ -1,13 +1,23 @@
-dist: xenial
-
-language: java
-jdk:
-  - openjdk11
-
-script:
-  - ./gradlew clean check
-
-branches:
-  only:
-    - master
-    - /^IO2019TEAM-.*$/
+matrix:
```

Comment:
To nie wiem, czy lepiej `matrix`, czy `jobs` - `matrix` brzmi jak na przykład testowanie tego samego builda na różnych wersjach języka/bibliotek/systemu

Tych wspólnych rzeczy typu `dist` i `branches` nie da się wyjąć?